### ISSUE TITLE
Home overflow menus don't inflate up when towards bottom of the screen

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#986]
## Steps to reproduce
1. Turn the device in landscape mode.
2. Open a session's overflow menu.
### Expected behavior
The menu is displayed above the fold.
### Actual behavior
The menu is displayed under the fold, you have to pull it up to see it. Scrolling the page down doesn't work.
Turning the device to portrait mode, the session title will disappear.
![20190313_112142](https://user-images.githubusercontent.com/4037709/54267635-42c85100-4582-11e9-8e2b-47a10976f458.gif)

### Device information

* Android device: Pixel (Android 9), Nexus 5(Android 6)
* Fenix version: master build 3/13.

**Comment by jatinkumarg** [comment]
Hey, just want to add more info about this bug. For me, on my device, it also removes the Archive button whenever I switch from portrait to landscape or vice versa

Potrait
![potrait](https://user-images.githubusercontent.com/20503830/54503199-85808500-4904-11e9-90fe-1f56344ebadd.png)


Landscape
![landscape](https://user-images.githubusercontent.com/20503830/54503198-85808500-4904-11e9-899e-ddea31cec6ef.png)

**Comment by topotropic** [comment]
We got rid of this menu view with introduction of the new Collections features but if you tap on the overflow menu now it still renders below the fold.

**Comment by sblatz** [comment]
@topotropic can you clarify what you mean?

I'm not able to reproduce this bug.

**Comment by topotropic** [comment]
See screenshot but I guess that's a rare case
![Screenshot (May 16, 2019 10:35:18 AM)](https://user-images.githubusercontent.com/353960/57839287-b5de8580-77c6-11e9-9021-eb1a96e93578.png)

**Comment by sblatz** [comment]
This is actually blocked on https://github.com/mozilla-mobile/android-components/issues/2916.

@vesta0, @bifleming can we get that a-c ticket properly prioritized?

**Comment by Amejia481** [comment]
@sblatz  We [could detect when the phone is in landscape](https://stackoverflow.com/questions/5112118/how-to-detect-orientation-of-android-device) and use if so pass an up orientation. Could this be an alternative?
 `orientation = BrowserMenu.Orientation.UP`


![Screenshot_20190531-170302](https://user-images.githubusercontent.com/773158/58735229-c2b6c800-83c7-11e9-9663-7998e905acc9.png). Could this be an alternative?

**Comment by softvision-miralobontiu** [comment]
This is not fixed yet. The menu is not entirely visible, it is scrollable though - please see the attached screenshot.
Tested with the latest build 1,0,1924 (#11630619), with OnePlus 5T (Android 9), Motorola Nexus 6 (Android 7,1,1).

![2019_06_12_15_49_37](https://user-images.githubusercontent.com/40694158/59352830-48bd0200-8d2a-11e9-8afb-def446fc29c6.gif)

**Comment by Amejia481** [comment]
I think this is a different issue from the original one. This only happens after the menu is opened on portrait mode, and while is open the phone is switched to landscape mode.

**Comment by softvision-miralobontiu** [comment]
Arturo, the same behavior is when opening that menu directly in landscape mode.

**Comment by sblatz** [comment]
I agree with @Amejia481 this seems like a separate issue. When I open the menu in landscape, it properly inflates upwards:
![demo](https://user-images.githubusercontent.com/4400286/59364411-a8e08380-8ceb-11e9-971c-f0d8e21679ed.gif)


@softvision-miralobontiu are you not seeing the menu inflate upwards in landscape?

I can file the rotation issue as a separate ticket :)

**Comment by sblatz** [comment]
Follow up issue filed at https://github.com/mozilla-mobile/fenix/issues/3390

**Comment by softvision-miralobontiu** [comment]
@sblatz in landscape the menu inflates, as you've mentioned.

I will close this as fixed.
